contracts/LpSugar.vy

```diff
@@ -132,6 +183,23 @@ interface IGauge:
   def rewardRateByEpoch(_ts: uint256) -> uint256: view
   def rewardToken() -> address: view
 
+interface ICLGauge:
+  def earned(_account: address, _position_id: uint256) -> uint256: view
+  def rewardRate() -> uint256: view
+  def rewardRateByEpoch(_ts: uint256) -> uint256: view
+  def rewardToken() -> address: view
+  def feesVotingReward() -> address: view
+  def stakedContains(_account: address, _position_id: uint256) -> bool: view
+
+interface IUniversalGauge: # fetches rewards data from both v2 and v3 gauges
+  def rewardRate() -> uint256: view
+  def rewardRateByEpoch(_ts: uint256) -> uint256: view
+  def rewardToken() -> address: view
```

Comment:
I suggest we merge these three interfaces into a single one and just mention in a comment which functions are specific to CL or v2.

Reply:
Ah, you're right. Let's keep the two separate. Still, `IUniversalGauge` seems like a duplicate of the `IGauge`, no?

Reply:
Ok, but `IUniversalGauge` doesn't seem to have any specific call. I'm just saying we can reuse any of the existing interfaces where that interface is used currently.